src/docs/product/accounts/sso/azure-sso.mdx

```diff
@@ -8,41 +8,47 @@ description: "Learn about the Azure Active Directory Single Sign-On (SSO)."
 
-3. In the search bar, search for "Groups" and navigate to "Groups." From there, create a new group, add an owner, and members to the group. Set "Group type" to Office 365. For more details about group creation, see the [Azure docs](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-groups-create-azure-portal).
+1. If you don't require a permission group for Sentry, skip to Step 5.
 
-4. Return to the Overview page. In the search bar, search for "Enterprise Applications" and navigate to "Enterprise Applications." Click "+ New application". Then, select "Non-gallery application" and create a non-gallery application for Sentry. Once created, you'll be directed to "Sentry - Overview.
+1. In the search bar, search for "Groups" and navigate to it. From there, create a new group, add an owner, and members to the group. Set "Group type" to Office 365. For more details about group creation, see the [Azure docs](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-groups-create-azure-portal).
+
+1. Return to the Overview page. In the search bar, search for "Enterprise Applications" and navigate to it. Click `+ New application`. Then, search for "Sentry" and create the application for Sentry. 
+   
+   ![Sentry in Azure Gallery](azure-search-sentry.png)
+
+1. Once created, you'll be directed to `Sentry - Overview`.
 
    ![Sentry Overview](azure-sentry-overview.png)
 
-5. Click on `1. Assign users and groups`, with "+ Add user" add yourself and the group you've created to the Sentry app. Click "Assign".
+1. Click on `1. Assign users and groups`, then `+ Add user`. Add yourself and the group you've created to the Sentry app. Click `Assign`.
 
-6. Navigate back to "Overview," click on `2. Set up single sign-on` and then select SAML as your single sign-on method.
+1. Navigate back to "Overview," click on `2. Set up single sign-on` and then select SAML as your single sign-on method.
 
-7. For Section (1) labeled as `Basic SAML Configuration`, enter the following data in each line and save your changes.
+1. For Section (1) labeled as `Basic SAML Configuration`, enter the following data in each line and save your changes.
 
-   - Identifier (entity ID): `https://sentry.io/saml/metadata/YOUR-ORG-SLUG/`
+   - Identifier (Entity ID): `https://sentry.io/saml/metadata/YOUR_ORG_SLUG/`
 
-   - Reply URL (Assertion Consumer Service URL): `https://sentry.io/saml/acs/YOUR-ORG-SLUG/`
+   - Reply URL (Assertion Consumer Service URL): `https://sentry.io/saml/acs/YOUR_ORG_SLUG/`
 
-   - Sign on URL: `https://sentry.io/auth/login/YOUR-ORG-SLUG/`
+   - Sign on URL: `https://sentry.io/auth/login/YOUR_ORG_SLUG/`
 
-   - Relay State: `https://sentry.io/YOUR-ORG-SLUG/`
+   - Relay State: `https://sentry.io/YOUR_ORG_SLUG/`
 
-   - Logout URL: `https://sentry.io/saml/sls/YOUR-ORG-SLUG/`
+   - Logout URL: `https://sentry.io/saml/sls/YOUR_ORG_SLUG/`
```

Comment:
This is better UX. 

`YOUR-ORG-SLUG` selects it word-by-word (user need to click/press delete 3 times)
`YOUR_ORG_SLUG` selects the entire block (can do it with 1 action).